cloudinary/models.py

```diff
@@ -106,7 +131,8 @@ def pre_save(self, model_instance, add):
         value = super(CloudinaryField, self).pre_save(model_instance, add)
         if isinstance(value, UploadedFile):
             options = {"type": self.type, "resource_type": self.resource_type}
-            options.update(self.upload_options_with_filename(model_instance, value.name))
+            options.update(self.options)
+            #options.update(self.upload_options_with_filename(model_instance, value.name))
```

Comment:
Please remove this, do not leave unused(commented out) code and all irrelevant functions related to it